static/app/components/pageTimeRangeSelector.tsx

```diff
@@ -1,66 +1,30 @@
-import {useState} from 'react';
 import styled from '@emotion/styled';
 
 import TimeRangeSelector from 'sentry/components/organizations/timeRangeSelector';
 import {Panel} from 'sentry/components/panels';
 import {DEFAULT_RELATIVE_PERIODS} from 'sentry/constants';
 import {t} from 'sentry/locale';
-import space from 'sentry/styles/space';
 
 type Props = React.ComponentProps<typeof TimeRangeSelector> & {
   className?: string;
 };
 
 function PageTimeRangeSelector({className, ...props}: Props) {
-  const [isCalendarOpen, setIsCalendarOpen] = useState(false);
-
   return (
-    <DropdownDate className={className} isCalendarOpen={isCalendarOpen}>
+    <DropdownDate className={className}>
       <TimeRangeSelector
         key={`period:${props.relative}-start:${props.start}-end:${props.end}-utc:${props.utc}-defaultPeriod:${props.defaultPeriod}`}
         label={<DropdownLabel>{t('Date Range:')}</DropdownLabel>}
-        onToggleSelector={isOpen => setIsCalendarOpen(isOpen)}
         relativeOptions={DEFAULT_RELATIVE_PERIODS}
         {...props}
       />
     </DropdownDate>
   );
 }
 
-const DropdownDate = styled(Panel)<{isCalendarOpen: boolean}>`
```

Comment:
we still might need to preserve some of these styles for the standalone date filter on release details and team stats
<img width="1201" alt="image" src="https://user-images.githubusercontent.com/9372512/157344645-dad3eaf4-5749-4c23-9ac7-863a6092d1fc.png">

Reply:
fixed in https://github.com/getsentry/sentry/pull/32349/commits/e0a40ac3c351d558a37f0b11a81528f52ce86664